healthscope/README.md

```diff
@@ -0,0 +1,197 @@
+# Health Scope Controller
+
+Health Scope Controller used for periodically check health scope crd and check the health of all the components related.
+
+## What will health scope controller do?
+
+1. periodically check all component health and update the CR status.
+2. serve as a http server, to output aggregated health information.
+
+## How to install?
+
+You could use helm to install it:
+
+```shell script
```

Comment:
```suggestion
```shell
```

Reply:
my IDE auto complete `shell` to `shell script `